backstop/tests/header.html

```diff
@@ -104,86 +112,122 @@ <h1>Header</h1>
+        Default
+        <label
+          class="iui-toggle-switch demo-disabled-toggle"
+          style="display: inline-flex; vertical-align: middle; margin-left: 1em;"
+        >
+          <input type="checkbox" />
+          <span class="iui-toggle">
+            <span class="iui-handle"></span>
+          </span>
+          <span class="iui-label">Disabled</span>
+        </label>
+      </h2>
       <section id="demo-default">
 
         <header class="iui-page-header">
-          <div class="iui-left">
+          <div class="iui-page-header-left">
             <!-- Application Button -->
             <div
-              class="iui-header-logo"
+              class="iui-header-brand"
               role="button"
               aria-label="App name"
               aria-expanded="false"
               tabindex="0"
             >
               <svg-imodel-hollow
-                class="iui-header-button-icon"
+                class="iui-header-brand-icon"
                 aria-hidden="true"
               ></svg-imodel-hollow>
-              <span class="iui-label">App name</span>
+              <span class="iui-header-brand-label">App name</span>
             </div>
 
-            <div class="iui-divider"></div>
+            <div class="iui-page-header-divider"></div>
 
             <nav aria-label="breadcrumbs">
               <!-- Project Button -->
-              <button
-                class="iui-button iui-borderless iui-header-button"
-                type="button"
-                aria-label="Project picker"
-                aria-expanded="false"
-                id="test-button-1"
-              >
-                <div class="iui-button-icon iui-header-button-icon demo-picture"></div>
-                <span class="iui-button-label">
-                  <div>Project Alpha</div>
-                  <div class="iui-description">0x0123456789</div>
-                </span>
-                <svg-caret-down-small
-                  class="iui-button-icon"
-                  aria-hidden="true"
-                ></svg-caret-down-small>
-              </button>
+              <span class="iui-header-breadcrumb">
+                <button
+                  class="iui-header-breadcrumb-button"
+                  type="button"
+                  aria-label="Project picker"
+                  aria-expanded="false"
+                  id="test-button-1"
+                >
+                  <img
+                    class="iui-header-breadcrumb-button-icon"
+                    src="../assets/image-test.jpg"
+                    draggable="false"
+                  />
+                  <span class="iui-header-breadcrumb-button-label">
+                    <div class="iui-header-breadcrumb-button-label-name">Project Alpha</div>
+                    <div class="iui-header-breadcrumb-button-label-description">0x0123456789</div>
+                  </span>
+                  <svg-caret-down-small
+                    class="iui-button-icon"
```

Comment:
This one feels random. Could we avoid this `iui-button-icon` class if we are not using `iui-button` anywhere?